### ISSUE TITLE
Cannot find module node_modules/my-timezone/dist/esm/index.js

### ISSUE THREAD

**Issue by wickedest** [ffflorian/node-packages#373]
I'm trying to use your module (thanks for it btw, it's exactly what I need). However, when trying to use it, I get:

```
Cannot find module 'node_modules/my-timezone/dist/esm/index.js' imported from src/index.js
```

There is no `index.js` in the dist:

```
node_modules/my-timezone/dist/
node_modules/my-timezone/dist/esm
node_modules/my-timezone/dist/esm/MyTimezone.js
node_modules/my-timezone/dist/esm/package.json
node_modules/my-timezone/dist/esm/cli.js
node_modules/my-timezone/dist/esm/MyTimezone.d.ts
node_modules/my-timezone/dist/esm/MyTimezone.test.d.ts
node_modules/my-timezone/dist/esm/MyTimezone.test.js
node_modules/my-timezone/dist/esm/cli.d.ts
node_modules/my-timezone/dist/cjs
node_modules/my-timezone/dist/cjs/MyTimezone.js
node_modules/my-timezone/dist/cjs/package.json
node_modules/my-timezone/dist/cjs/cli.js
node_modules/my-timezone/dist/cjs/MyTimezone.d.ts
node_modules/my-timezone/dist/cjs/MyTimezone.test.d.ts
node_modules/my-timezone/dist/cjs/MyTimezone.test.js
node_modules/my-timezone/dist/cjs/cli.d.ts
```

**Comment by ffflorian** [comment]
Hi @wickedest, thank you for your report. I realized I never used it as a module, I always used the CLI. 😃

Anyway, I just created a release and this should be fixed with `my-timezone@1.4.9`.

**Comment by wickedest** [comment]
thanks @ffflorian , but it still has an issue:

```
Cannot find package 'axios' imported from ./node_modules/my-timezone/dist/esm/MyTimezone.js
```

**Comment by ffflorian** [comment]
@wickedest this should be fixed with `my-timezone@1.5.1` 🙂
